modules/rubiconBidAdapter.js

```diff
@@ -150,14 +139,14 @@ export const spec = {
       let slotData = {
         site_id: params.siteId,
         zone_id: params.zoneId,
-        position: parsePosition(params.position),
+        position: params.position === 'atf' || params.position === 'btf' ? params.position : 'unknown',
         floor: parseFloat(params.floor) > 0.01 ? params.floor : 0.01,
         element_id: bidRequest.adUnitCode,
         name: bidRequest.adUnitCode,
         language: params.video.language,
         width: size[0],
         height: size[1],
-        size_id: params.video.size_id
+        size_id: utils.deepAccess(bidRequest, `mediaTypes.${VIDEO}.context`) === 'outstream' ? 203 : params.video.size_id
```

Comment:
We are setting the `size_id` to be `203` no matter what if it is outstream,

Above at line 102 we reject a video bid request if it does not contain `params.video.size_id`

So are we requiring that anyone who uses outstream with us place `size_id` in the video params field?

Looks to me like if we are just always changing it to `203` ourselves, we should not have this strict check above?